contrib/scripts/kind.sh

```diff
@@ -150,15 +161,20 @@ kubeadmConfigPatches:
         "v": "3"
 EOF
 
+nodes=$(docker ps -a --filter label=io.x-k8s.kind.cluster=${cluster_name:-kind} --format {{.Names}})
+for node in $nodes; do
+  docker network connect ${default_secondary_network} $node
+done
+
 if [ "${xdp}" = true ]; then
   if ! [ -f "${CILIUM_ROOT}/test/l4lb/bpf_xdp_veth_host.o" ]; then
     pushd "${CILIUM_ROOT}/test/l4lb/" > /dev/null
     clang -O2 -Wall -target bpf -c bpf_xdp_veth_host.c -o bpf_xdp_veth_host.o
     popd > /dev/null
   fi
 
-  for ifc in /sys/class/net/"${bridge_dev}"/brif/*; do
-    ifc="${ifc#"/sys/class/net/${bridge_dev}/brif/"}"
+  for ifc in /sys/class/net/"${bridge_dev}"*/brif/*; do
+    ifc=$(echo $ifc | sed "s:/sys/class/net/${bridge_dev}.*/brif/::")
```

Comment:
let's use `$SED` here, which should point to GNU sed on macos too

Reply:
Nice catch! Thanks